bin/gwdetchar-lasso-correlation

```diff
@@ -99,20 +100,15 @@ parser.add_argument('-J', '--nproc-plot', type=int, default=None,
 parser.add_argument('-o', '--output-dir', default=os.curdir,
                     help='output directory for plots')
 parser.add_argument('-f', '--channel-file', type=os.path.abspath,
-                    help='path for channel file')
+                    help='path for channel file', required=True)
```

Comment:
@macedo22, I'm not a fan of required optional arguments, since that makes no sense? Can you have a think about changing this to a positional argument if it really is `required`?

Reply:
Yes, I can! I can just remove it to keep options open, as you say